components/channel_members_rhs/search.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import styled from 'styled-components';
+
+import Input from '../widgets/inputs/input/input';
+
+interface Props {
+    className?: string;
+    terms: string;
+    onInput: (terms: string) => void;
+}
+
+const SearchBar = ({className, terms, onInput}: Props) => {
+    const {formatMessage} = useIntl();
+    return (
+        <div className={className}>
+            <Input
+                value={terms}
+                onInput={(ev) => onInput(ev.currentTarget.value)}
+                inputPrefix={<i className={'icon icon-magnify'}/>}
+                inputSuffix={terms.length > 0 ? (
+                    <button
+                        className='style--none'
+                        onClick={() => onInput('')}
+                        aria-label={formatMessage({
+                            id: 'channel_members_rhs.search_bar.aria.cancel_search_button',
+                            defaultMessage: 'cancel members search',
+                        })}
+                    >
+                        <i className={'icon icon-close-circle'}/>
+                    </button>
+                ) : <></>}
```

Comment:
I think it is better to set null instead of an empty fragment.

Reply:
Thanks for the feedbacks!